apps/mobile/src/hooks/useSign/useSign.ts

```diff
@@ -15,7 +15,7 @@ export const storePrivateKey = async (userId: string, privateKey: string) => {
       invalidateOnNewBiometry: true,
     })
 
-    const encryptyedPrivateKey = await DeviceCrypto.encrypt(userId, privateKey, {
+    const encryptedPrivateKey = await DeviceCrypto.encrypt(userId, privateKey, {
```

Comment:
just a nit: Those changes should have been a separate PR. The implications of this change mostly later in the const {encryptedPassword, iv } ... is that previously stored private keys are no longer properly decrypted :) When we merge this PR it won't be obvious why a previously imported PK would no longer work since there is no commit that does anything with the PK.